### PR TITLE
Update rockybala.yml

### PR DESCRIPTION
**Pull request by rockybala**
Added a presentation

### CODE REVIEW

**Comment by rockybala** [comment]
Hi @pelmer, Could you please merge this file into master. I have updated one of my talks..

**Comment by henryiii** [comment]
It's invalid YAML. You can't have an unquoted colon unless it is not proceeded by a space (basically, URLs are okay, anything else has to be quoted)

File: _data/people/rockybala.yml [hunk 1]
```diff
@@ -32,3 +32,11 @@ presentations:
   location: CERN
   project: acts
   focus-area: ia
+- title: Qualification task in TrackingCP: ACTS Integration into Athena
```
Comment (henryiii) [inline _data/people/rockybala.yml]:
```suggestion
- title: "Qualification task in TrackingCP: ACTS Integration into Athena"
```

**Comment by henryiii** [comment]
Please click on "details" and look through the logs, the problem is usually printed out. Or bug one of us, that's fine too, I'm always happy to fix things. :)

**Comment by rockybala** [comment]
Hi @henryiii, I have updated the file. Could you merge it now?

**Comment by henryiii** [comment]
The project needs to be one of the known projects, "acts" in this case, it's not an arbitrary string. I also already added quotes around the title, which is all that's needed. YAML will turn `a: b c: d` into `"a": {"b c": "d"}}`, so you have to add the quotes `a: "b c: d"` to get the correct `{"a": "b c: d"}` result.

I've corrected the project, and am merging now that the CI has passed.

**Comment by henryiii** [comment]
Changes will show up on the website in about 3-5 minutes.

**Pull request merged**